Allow rank computation for QRPivoted matrices (#54283)
JuliaLang/julia: NEWS.md

#### LinearAlgebra

> * `rank` can now take a QRPivoted matrix ([#53214]) to allow rank computation on QR-factorized matrices.

Reviewer: The news item should cross reference the PR, not the issue.

Author: My bad, it should be ok now

> * `rank` can now take a QRPivoted matrix ([#53214]) to allow rank computation on QR-factorized matrices.

Reviewer 2: ```suggestion
* `rank` can now take a `QRPivoted` matrix to allow rank estimation via QR factorization ([#53214]).
```

Author: Done, LMK whether I should merge after this
